core/src/com/biglybt/plugin/net/buddy/BuddyPlugin.java

```diff
@@ -1,2242 +1,2243 @@
-		return( false );
-	}
-	
-	public BuddyPluginBuddy
-	addBuddy(
-		boolean		public_network,
-		String		key,
-		int			subsystem )
-	{
-		BuddyPluginNetwork	net = getPluginNetwork( public_network );
-		
-		if ( net != null ){
-			
-			return( net.addBuddy(key, subsystem));
-		}
-		
-		return( null );
-	}
-	
-	public BuddyPluginBuddy
-	peekBuddy(
-		boolean		public_network,
-		String		key )
-	{
-		BuddyPluginNetwork	net = getPluginNetwork( public_network );
-		
-		if ( net != null ){
-		
-			return( net.peekBuddy(key));
-		}
-		
-		return( null );
-	}
-	
-	public List<BuddyPluginBuddy>
-	getBuddies()
-	{
-		if ( plugin_networks.length == 1 ){
-			
-			return( plugin_networks[0].getBuddies());
-			
-		}else{
-			
-			List<BuddyPluginBuddy>	result = new ArrayList<>();
-			
-			for ( BuddyPluginNetwork pn: plugin_networks ){
-				
-				result.addAll( pn.getBuddies());
-			}
-			
-			return( result );
-		}
-	}
-	
-	public BuddyPluginBuddy
-	getBuddyFromPublicKey(
-		String		key )
-	{
-		for ( BuddyPluginNetwork pn: plugin_networks ){
-			
-			BuddyPluginBuddy buddy = pn.getBuddyFromPublicKey(key);
-			
-			if ( buddy != null ){
-				
-				return( buddy );
-			}
-		}
-		
-		return( null );
-	}
-	
-		// ******
-	
-	public void
-	logMessage(
-		BuddyPluginBuddy	buddy,
-		String				str,
-		Throwable			e )
-	{
-		logMessage( buddy, str + ": " + Debug.getNestedExceptionMessage(e), true );
-	}
-
-	public void
-	logMessage(
-		BuddyPluginBuddy	buddy,
-		String				str )
-	{
-		logMessage( buddy, str, false );
-	}
-
-	public void
-	logMessage(
-		BuddyPluginBuddy	buddy,
-		String				str,
-		boolean				is_error )
-	{
-		if ( buddy != null && ( buddy.isTransient() || ! buddy.isAuthorised())){
-			
-			return;
-		}
-		
-		log( buddy, str );
-
-		Iterator it = listeners.iterator();
-
-		while( it.hasNext()){
-
-			try{
-				((BuddyPluginListener)it.next()).messageLogged( str, is_error );
-
-			}catch( Throwable e ){
-
-				Debug.printStackTrace( e );
-			}
-		}
-	}
-
-	public void
-	log(
-		BuddyPluginBuddy	buddy,
-		String				str )
-	{
-		if ( buddy != null && ( buddy.isTransient() || ! buddy.isAuthorised())){
-			
-			return;
-		}
-
-		logger.log( str );
-	}
-
-	public void
-	log(
-		BuddyPluginBuddy	buddy,
-		String				str,
-		Throwable			e )
-	{
-		if ( buddy != null && ( buddy.isTransient() || ! buddy.isAuthorised())){
-			
-			return;
-		}
-		
-		logger.log( str + ": " + Debug.getNestedExceptionMessageAndStack( e ));
-	}
-	
-	public interface
-	CryptoResult
-	{
-		public byte[]
-		getChallenge();
-
-		public byte[]
-		getPayload();
-	}
-	
-	protected static class
-	FeedDetails
-	{
-		private byte[]		contents;
-		private String		last_modified;
-
-		protected
-		FeedDetails(
-			byte[]		_contents,
-			String		_last_modified )
-		{
-			contents		= _contents;
-			last_modified	= _last_modified;
-		}
-
-		protected byte[]
-		getContent()
-		{
-			return( contents );
-		}
-
-		protected String
-		getLastModified()
-		{
-			return( last_modified );
-		}
-	}
-}
+/*
```

Comment:
My intellij shows only one added line and one changed line. 
I don't see whats wrong. Please advice!

Reply:
Here's my diff 
![bilde](https://user-images.githubusercontent.com/13136714/54887154-eae9ee00-4e8f-11e9-8586-f9fefa8746ad.png)

Reply:
Seems like that file has CRLF style of line endings. 
This is what i see opening the raw file at https://raw.githubusercontent.com/BiglySoftware/BiglyBT/master/core/src/com/biglybt/plugin/net/buddy/BuddyPlugin.java

![bilde](https://user-images.githubusercontent.com/13136714/54950863-8fc50380-4f42-11e9-8abf-49d249d19a01.png)

Reply:
I created a PR to fix the line endings in this repo. See #965

Reply:
I re-added CRLF to spesific file as a workaround for this PR.

Reply:
@parg - I suggest you set the autocrlf flag, Heres the command:
```
git config --global core.autocrlf true
```